cogboard-webapp/src/components/MainTemplate/styled.js

```diff
@@ -50,3 +52,15 @@ export const StyledDrawer = styled(props => (
     background: ${getBackgroundColor('default')};
   }
 `;
+
+export const drawerStyles = makeStyles({
+  root: {
+    justifyContent: 'space-between'
+  }
+});
```

Comment:
You can access the root using styled like this `styled(props => <Component classes={{root: 'root'}} {...props} />)` root: styles... ``